settings.py

```diff
@@ -1186,18 +1192,16 @@ def JINJA_CONFIG():
         'kuma': {
             'handlers': ['console'],
             'propagate': True,
-            # Use the most permissive setting. It is filtered in the handlers.
-            'level': logging.DEBUG,
+            'level': logging.ERROR,
         },
         'django.request': {
             'handlers': ['console'],
             'propagate': True,
-            # Use the most permissive setting. It is filtered in the handlers.
-            'level': logging.DEBUG,
+            'level': logging.ERROR,
         },
         'elasticsearch': {
-            'level': logging.ERROR,
             'handlers': ['console'],
+            'level': logging.ERROR,
```

Comment:
I would love it if MDN had a logging facility set up so I can view the logs from the staging server. Then in my ES commit I would change this to DEBUG to check on this for a week or so.